resources/quality/xyzprinting/abs/xyzprinting_da_vinci_jr_pro_xplus_copper_0.40_abs_fine.inst.cfg

```diff
@@ -0,0 +1,32 @@
+[general]
+version = 4
+name = Fine Quality
+definition = xyzprinting_da_vinci_jr_pro_xplus
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = fine
+material = generic_abs
+variant = Copper 0.4mm Nozzle
+
+[values]
+cool_fan_speed = 100
+cool_fan_speed_0 = 0
+material_bed_temperature = 70
+material_bed_temperature_layer_0 = 70
+material_print_temperature = 240
```

Comment:
It's generally better to not set the material_print temperature in the quality profile. If you do that, changing the setting in the material profile won't work.

It's better to not set it all and use something like this for the other ones :
material_final_print_temperature = =material_print_temperature (and if that is always the case, it's also better to set that in the machine definition and not in a specific profile)

Reply:
Thanks for your advisement,it is more convenient :)

Reply:
Maybe for you, but for users it breaks their material profiles.